docs/officers/dues/data-model.rst

```diff
@@ -0,0 +1,49 @@
+Data Model
+==========
+
+Dues are tracked in Apiary using several related objects.
+
+-----------
+Fiscal Year
+-----------
+
+A **Fiscal Year** groups together several `Dues Package`_ and `Merchandise`_ objects.
+Fiscal years are primarily used for reporting and do not affect any functionality.
+They need to be created manually once a year to group the objects for that year.
+
+---------------
+_`Dues Package`
+---------------
```

Comment:
There should probably be a link here to the instructions page.

Reply:
I don't think that's necessary; there's already links in two other places from the navigation UIs.

Reply:
That's fair. I reviewed while test was down originally.